packages/core/src/components/MonthPicker/MonthPicker.test.jsx

```diff
@@ -237,7 +237,7 @@ describe('MonthPicker', () => {
 
   it('renders a snapshot', () => {
     const tree = renderer
-      .create(<MonthPicker name="months" label="Months" />)
+      .create(<MonthPicker name="months" label="Months" locale="en" />)
```

Comment:
Do you think we should do follow on work for other tests?

Reply:
If there are other components that use the `toLocaleString` function, then yes. You can audit it by doing a search for that string. I'm fairly certain we're not using it anywhere else

Reply:
Thanks @pwolfert it looks like Mask is also using `toLocaleString` https://github.com/CMSgov/design-system/search?q=toLocaleString&unscoped_q=toLocaleString should we change there too?

Reply:
Oh, and it's hard-coded. This ones a bit tricker. I think we should just circle back to it when it becomes a problem. It's not an actual issue in the real world—just in testing.